src/helpers/json.ts

```diff
@@ -0,0 +1,36 @@
+import { camelCase, snakeCase } from 'lodash';
```

Comment:
FYI, we can reduce the bundle size by 
```suggestion
import camelCase from 'lodash/camelCase';
import snakeCase from 'lodash/snakeCase';
```

Reply:
Thanks, it's useful